jobs/build_docker/get_docker_commit_version.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash 
+set -e
+
+build_record=`ls ${DOCKER_RECORD_PATH}`
+image_list=`head -n 1 $build_record`
+docker_repo_commit_file="${DOCKER_REPO_HASHCODE_FILE}"
+commit_string_file=commitstring.txt
+
+# global var
+on_core_hash=
+on_tasks_hash=
+hashcode=
+
+# basic check, clean env
+prepare_check_test(){
+    running_container="$(docker ps -q | wc -l)"
+    echo "[DEBUG] all running_container count:${running_container}"
+
+    time=1
+    MAX_TRY=10
+    while [ ${running_container} -lt 9 ] && [ ${time} -le ${MAX_TRY} ]; do
+        sleep 10
+        running_container="$(docker ps -q | wc -l)"
+        echo "[DEBUG] [${time}*10s] all running_container count:${running_container}"
+        time=`expr ${time} + 1`
+    done
+
+    if [ ${running_container} -lt 9 ]; then
+        echo "[ERROR] not all containers is running, only ${running_container}, need 9."
+        exit 1
+    fi
+    rm -rf ${docker_repo_commit_file}
+}
+    
+# clean up docker env: clean_docker_env ${repo_tag}
+clean_docker_env(){
+    container_id="$(docker ps -a -q --filter ancestor=$1 --format="{{.ID}}")"
+    for container in ${container_id}; do
+        echo "[DEBUG] docker rm container:${container}"
+        docker rm ${container}
+    done    
+}
+
+# is_core_correct ${hashcode}
+is_core_correct(){
+    if [ -z "${on_core_hash}" ]; then 
```

Comment:
do you need on_core_hash being pass(check in on-core) in before checking other repos?